libs/sdk-ui-tests/stories/visual-regression/kit/Overlay/Overlay.tsx

```diff
@@ -1,13 +1,15 @@
 // (C) 2007-2020 GoodData Corporation
 import React from "react";
 import { injectIntl, FormattedMessage, IntlShape, IntlProvider } from "react-intl";
-import MediaQuery from "react-responsive";
+import DefaultMediaQuery, { MediaQueryProps } from "react-responsive";
 import { storiesOf } from "../../../_infra/storyRepository.js";
 import { UiKit } from "../../../_infra/storyGroups.js";
 import { Button, Overlay, FullScreenOverlay } from "@gooddata/sdk-ui-kit";
 
 import "../styles/goodstrap.scss";
 
+const MediaQuery = DefaultMediaQuery as unknown as React.ComponentType<MediaQueryProps>;
```

Comment:
Not sure if this will work in runtime. We had with this library problems. And solved via default-import 


import DefaultMediaQuery from "react-responsive";
import { defaultImport } from "default-import";

// There are known compatibility issues between CommonJS (CJS) and ECMAScript modules (ESM).
// In ESM, default exports of CJS modules are wrapped in default properties instead of being exposed directly.
// https://github.com/microsoft/TypeScript/issues/49189#issuecomment-1137802865
const MediaQuery = defaultImport(DefaultMediaQuery);